code/_globalvars/lists/misc_lists.dm

```diff
@@ -61,5 +61,8 @@ GLOBAL_LIST_EMPTY(client_login_processors)
 // Assoc-List of all karma packages
 GLOBAL_LIST_EMPTY(karma_packages)
 
+// List of all possible vendor crits
+GLOBAL_LIST_EMPTY(vendor_crits)
+
```

Comment:
On second thought, can you make crits be stored in a static list on vendors instead of a global?

Reply:
Updated, let me know if my approach is along the lines you were thinking of

Reply:
LGTM.